language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -30,7 +30,8 @@ import Data.Maybe
 import Data.Bifoldable
 import Options.Applicative
 import System.Directory
-import System.FilePath hiding ((<.>))
+import System.FilePath hiding ((<.>), (</>))
+import System.FilePath.Posix((</>)) -- Make sure we generate / on all platforms.
```

Comment:
Can't we just import `System.FilePath.Posix` _instead_ of `System.FilePath`.

Reply:
i agree it seems sensible. see https://github.com/digital-asset/daml/pull/4963.

Reply:
Update : after discussion, it seems the safer choice is to not replace `FilePath` with `FilePath.Posix` so, it was decided to not land https://github.com/digital-asset/daml/pull/4963.